client/src/components/categorical/index.js

```diff
@@ -227,32 +176,33 @@ class Categories extends React.Component {
           }
         />
 
-        <div style={{ marginLeft: 0 }}>
-          {
-            this.maybeRenderSingleLabel(
-              allCategoryNames,
-              schema,
-              expandedCats,
-              createAnnoModeActive
-            ) /* Categories with only one label */
-          }
-        </div>
-        {
-          this.maybeRenderReadOnly(
-            allCategoryNames,
-            schema,
-            expandedCats,
-            createAnnoModeActive
-          ) /* Read only categorical fields, ostensibly 'normal' mode */
-        }
-        {
-          this.maybeRenderAnnotations(
-            allCategoryNames,
-            schema,
-            expandedCats,
-            createAnnoModeActive
-          ) /* Writable fields, ie., anno */
-        }
+        {/* READ ONLY CATEGORICAL FIELDS */}
```

Comment:
Why pass `isUserAnno` if the <Category> already knows about the schema `.writable`?   (see comment in category/index.js)